lib/SILGen/SILGenExpr.cpp

```diff
@@ -506,8 +506,13 @@ namespace {
     // SWIFT_ENABLE_TENSORFLOW
     RValue visitDifferentiableFunctionExpr(DifferentiableFunctionExpr *E,
                                            SGFContext C);
+    RValue visitLinearFunctionExpr(LinearFunctionExpr *E, SGFContext C);
     RValue visitDifferentiableFunctionExtractOriginalExpr(
         DifferentiableFunctionExtractOriginalExpr *E, SGFContext C);
+    RValue visitLinearFunctionExtractOriginalExpr(
+        LinearFunctionExtractOriginalExpr *E, SGFContext C);
+    RValue visitLinearToDifferentiableFunctionExpr(
+        LinearToDifferentiableFunctionExpr *E, SGFContext C);
```

Comment:
Minor: add `// SWIFT_ENABLE_TENSORFLOW END`

Reply:
Done in 2fe68a2.